<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptPar</class>
 <widget class="QDialog" name="OptPar">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Optimisation parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Differential Evolution</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="0">
        <widget class="QDoubleSpinBox" name="de_atol">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="prefix">
          <string>atol: </string>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QComboBox" name="de_initialisation">
         <property name="toolTip">
          <string>Initialisation method</string>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>latinhypercube</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>random</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="de_target">
         <property name="toolTip">
          <string>maximise log-likelihood or log-posterior probability</string>
         </property>
         <property name="currentText">
          <string>log-likelihood</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>log-likelihood</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>log-posterior</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="de_popsize">
         <property name="prefix">
          <string>popsize: </string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="de_maxiter">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="prefix">
          <string>maxiter: </string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QDoubleSpinBox" name="de_tol">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="prefix">
          <string>tol: </string>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.001000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="de_strategy">
         <property name="toolTip">
          <string>evolution strategy</string>
         </property>
         <property name="currentText">
          <string>best1bin</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>best1bin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>best1exp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>rand1exp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>randtobest1exp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>currenttobest1exp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>best2exp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>rand2exp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>randtobest1bin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>currenttobest1bin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>best2bin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>rand2bin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>rand1bin</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="de_recombination">
         <property name="prefix">
          <string>recombination: </string>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.700000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="de_mutation_lb">
         <property name="prefix">
          <string>mutation lower: </string>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="de_mutation_ub">
         <property name="prefix">
          <string>mutation upper: </string>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Levenberg Marquardt</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>L-BFGS-B</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Dual Annealing</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="da_target">
         <property name="toolTip">
          <string>maximise log-likelihood or log-posterior probability</string>
         </property>
         <property name="currentText">
          <string>log-likelihood</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>log-likelihood</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>log-posterior</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="da_maxiter">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="prefix">
          <string>maxiter: </string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QDoubleSpinBox" name="da_initial_temp">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="prefix">
          <string>initial temp: </string>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>50000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>5230.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="da_restart_temp">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="prefix">
          <string>restart temp ratio: </string>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000100000000000</double>
         </property>
         <property name="value">
          <double>0.000020000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QDoubleSpinBox" name="da_visit">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="prefix">
          <string>visit:</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>2.620000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QDoubleSpinBox" name="da_accept">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="prefix">
          <string>accept:</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-100000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>-5.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>-5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="da_no_local_search">
         <property name="text">
          <string>no local search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>SHGO</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>MCMC</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>de_target</tabstop>
  <tabstop>de_strategy</tabstop>
  <tabstop>de_maxiter</tabstop>
  <tabstop>de_popsize</tabstop>
  <tabstop>de_tol</tabstop>
  <tabstop>de_atol</tabstop>
  <tabstop>de_initialisation</tabstop>
  <tabstop>de_recombination</tabstop>
  <tabstop>de_mutation_lb</tabstop>
  <tabstop>de_mutation_ub</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptPar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptPar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
